README.md

```diff
@@ -10,6 +10,11 @@ The three vulnerable extensions reside in sub-folders of their own, and each suc
 
 For more background information and possible fixes for the vulnerabilities in the extensions of this repo, you can refer to the report we wrote (also part of this project). It is available in pdf format [here](TODO). (TODO: Add pdf of report in repo, and link to it here.)
 
+## How to run a node web server
+The websites included in this repo are all implemented using [node.js](https://nodejs.org/en/) and [express](https://expressjs.com/). To start the server for one of the websites, enter the folder of the website (for example [unsafe-fetch-vuln/web-server/](https://github.com/andreaskth/securing-chrome-extensions/tree/main/unsafe-fetch-vuln/web-server)). Run `npm install` to install the necessary dependencies. Then, in the same folder, run `node` followed by the name of the main file of the website to launch the application. All main files of the websites in this repo are called `app.js`, so you would run `node app.js`. The website should now be accesible in your browser if you go to `localhost:` followed by the port number of the application (the default port numbers are specifed in the README:s of the folders for the vulnerabilities). 
```

Comment:
node.js --> Node.js
express -> Express

"... Run `npm install` to install the necessary dependencies. Then, in the same folder, ..." --> "Run `npm install` to install the necessary dependencies (if it was not there before, you will see a new `node_modules` folder was added). Then, in the same folder,"
Could also leave this one as it is :)